### PR TITLE
youtube documentation update

### LINKED ISSUE #8667: [Docs]: Update the Documentation for the YouTube Integration
### Short description of what should be added or improved

The documentation for the YouTube handler needs to be updated in line with the recent updates that have been made, especially the use of OAuth for authentication.

### Video or screenshots

_No response_

### Anything else?

_No response_

### PR DESCRIPTION
**Pull request by QuantumPlumber**
## Description

Documentation update to reflect the YouTube handler Readme.md. I assume that the read me is up to date.

Fixes #8667 

## Type of change
- [ x ] 📄 This change requires a documentation update

## Checklist:

- [ x ] My code follows the style guidelines(PEP 8) of MindsDB.
- [ x ] I have appropriately commented on my code, especially in complex areas.
- [ x ] Necessary documentation updates are either made or tracked in issues.
- [ x ] Relevant unit and integration tests are updated or added.

### CODE REVIEW

File: docs/integrations/app-integrations/youtube.mdx [hunk 1]
```diff
@@ -5,66 +5,79 @@ sidebarTitle: YouTube
...
-[YouTube](https://www.youtube.com/) is a popular online video-sharing platform and social media website where users can upload, view, share, and interact with videos created by individuals and organizations from around the world.
+[YouTube](https://www.youtube.com/) is a popular online video-sharing platform and social media website where users
+can upload, view, share, and interact with videos created by individuals and organizations from around the world.
 
-Data from YouTube can be utilized within MindsDB to train AI models and chatbots based on user comments, detect sentiment of comments, or use AI models to create replies.
+Comment Data from YouTube can be utilized within MindsDB to train AI models and chatbots based on user comments,
+detect sentiment of comments, or use AI models to create replies.
 
-## Connection
+## Install the Handler
```
Comment (martyna-mindsdb) [inline docs/integrations/app-integrations/youtube.mdx:14]: Throughout the app integration docs, we use sections "Connection" and "Usage".  Let's keep it consistent.
So let's change it back to Connection.

File: docs/integrations/app-integrations/youtube.mdx [hunk 2]
```diff
@@ -5,66 +5,79 @@ sidebarTitle: YouTube
...
 
 The first step is to create a database with the new `Youtube` engine using the following command:
 
 ```sql
 CREATE DATABASE mindsdb_youtube
-WITH ENGINE = 'youtube',
+WITH ENGINE = 'YouTube',
 PARAMETERS = {
-  "youtube_api_token": "<your-youtube-api-key-token>"  
+  "YouTube_api_token": "<your-YouTube-api-key-token>"
```
Comment (martyna-mindsdb) [inline docs/integrations/app-integrations/youtube.mdx:39]: Same as above - this should be `youtube_api_token`.

File: docs/integrations/app-integrations/youtube.mdx [hunk 3]
```diff
@@ -5,66 +5,79 @@ sidebarTitle: YouTube
...
 
-Use the established connection to query the `get_comments` table for any YouTube Video of your choice.
-
-```sql
-SELECT * FROM mindsdb_youtube.get_comments
-WHERE youtube_video_id = "raWFGQ20OfA";
-```
+Use the established connection to query the comments table
 
+~~~~sql
```
Comment (martyna-mindsdb) [inline docs/integrations/app-integrations/youtube.mdx:52]: In the MDX format, we use triple backticks, so please change it back throughout this doc page.

**Review by MinuraPunchihewa — CHANGES_REQUESTED** [review]
Hey Ave!
Thank you for this. There is a small update that we need to make to the Connection section, specifically to the parameters that are used to initialize the handler. It is now possible to pass OAuth credentials (generated through the GCP console) as a URL, file path and even upload it directly (when using the form).

P.S: You have the coolest username!

**Comment by martyna-mindsdb** [comment]
Hey @MinuraPunchihewa @QuantumPlumber 
I'm waiting until the credentials_file param is supported, and also, until OAuth2 (credentials_url) is deployed.
Then, I plan to update yt docs to include OAuth2.

File: docs/integrations/app-integrations/youtube.mdx [hunk 4]
```diff
@@ -5,66 +5,81 @@ sidebarTitle: YouTube
...
 
 ## Connection
 
-This handler is implemented using the `google-api-python-client` library, a Python library that from where we can use YouTube v3 API.
+First, we have to install the required python packages.
+```bash
+pip install mindsdb[youtube]
+```
+
+## Usage
```
Comment (martyna-mindsdb) [inline docs/integrations/app-integrations/youtube.mdx:21]: Please remove this "## Usage" from this line.

The Connection section ends after the CREATE DATABASE statement. And there is already the Usage section.

File: docs/integrations/app-integrations/youtube.mdx [hunk 5]
```diff
@@ -5,66 +5,81 @@ sidebarTitle: YouTube
...
 ## Usage
 
-Use the established connection to query the `get_comments` table for any YouTube Video of your choice.
+Use the established connection to query the comments table
 
-```sql
-SELECT * FROM mindsdb_youtube.get_comments
-WHERE youtube_video_id = "raWFGQ20OfA";
-```
+~~~~sql
```
Comment (martyna-mindsdb) [inline docs/integrations/app-integrations/youtube.mdx:54]: Please change it back to triple backticks as above.

File: docs/integrations/app-integrations/youtube.mdx [hunk 6]
```diff
@@ -5,66 +5,81 @@ sidebarTitle: YouTube
...
+~~~~
 
+Advanced queries for the YouTube handler
 
-Advanced queries which are supported
-
-```sql
-SELECT * FROM mindsdb_youtube.get_comments
-WHERE youtube_video_id = "raWFGQ20OfA"
+~~~~sql
```
Comment (martyna-mindsdb) [inline docs/integrations/app-integrations/youtube.mdx:61]: Please change it back to triple backticks as above.

File: docs/integrations/app-integrations/youtube.mdx [hunk 7]
```diff
@@ -5,66 +5,81 @@ sidebarTitle: YouTube
...
 ORDER BY display_name ASC
 LIMIT 5;
-```
+~~~~
 
+Given a channel_id, get information about the channel
 
-Get information about any youtube video using video_id:
-```sql
+~~~~sql
```
Comment (martyna-mindsdb) [inline docs/integrations/app-integrations/youtube.mdx:70]: Please change it back to triple backticks as above.

File: docs/integrations/app-integrations/youtube.mdx [hunk 8]
```diff
@@ -5,66 +5,81 @@ sidebarTitle: YouTube
...
+SELECT * FROM mindsdb_youtube.channels
+WHERE channel_id="UC-...";
+~~~~
+
+Here, ```channel_id``` column is mandatory in the where clause.
+
+Get information about any YouTube video using video_id:
+
+~~~~sql
```
Comment (martyna-mindsdb) [inline docs/integrations/app-integrations/youtube.mdx:79]: Please change it back to triple backticks as above.

File: docs/integrations/app-integrations/youtube.mdx [hunk 9]
```diff
@@ -5,66 +5,81 @@ sidebarTitle: YouTube
...
+* ```youtube_api_token``` which is a Google API key used for authentication.
 
 <Tip>
-Check out [this guide](https://blog.hubspot.com/website/how-to-get-youtube-api-key) on how to create the API key in order to access YouTube data.
+Check out [this guide](https://blog.hubspot.com/website/how-to-get-youtube-api-key) on how to create the API key in
+    order to access YouTube data.
 </Tip>
 
-The first step is to create a database with the new `Youtube` engine using the following command:
+The first step is to create a database with the new ```youtube``` engine using the following command:
```
Comment (martyna-mindsdb) [inline docs/integrations/app-integrations/youtube.mdx:34]: The inline code uses single backticks. So it should be "...with the new \`youtube` engine..."

**Review by martyna-mindsdb — APPROVED** [review]

**Review by MinuraPunchihewa — APPROVED** [review]

**Pull request merged**